package/src/utils/utils.ts

```diff
@@ -564,3 +564,10 @@ export const reduceMessagesToString = <
 >(
   messages: FormatMessageResponse<StreamChatGenerics>[],
 ): string => messages.map(stringifyMessage).join();
+
+/** RegEx for special characters in string*/
+const specialCharacterRegex = /[|&;$%@"<>()+,]/g;
+
+/**Function to replace special characters in a string with '' */
+export const replaceSpecialCharacters = (value: string): string =>
+  value.replace(specialCharacterRegex, '');
```

Comment:
What makes a character special? Where is this used? Can we add some more context to the names here? The comments aren't necessary if the naming is clearer (and currently don't add any extra information)

Does it make sense to put this closer to where it's being used? `utils` is a bit vague, we should rather be clearer about the usage if possible

Reply:
True!!

Reply:
Actually doesn't need to be a separate a function!!